includes/admin/list-tables/class-wc-admin-list-table-coupons.php

```diff
@@ -47,6 +47,7 @@ protected function render_blank_state() {
 		echo '<div class="woocommerce-BlankState">';
 		echo '<h2 class="woocommerce-BlankState-message">' . esc_html__( 'Coupons are a great way to offer discounts and rewards to your customers. They will appear here once created.', 'woocommerce' ) . '</h2>';
 		echo '<a class="woocommerce-BlankState-cta button-primary button" target="_blank" href="https://docs.woocommerce.com/document/coupon-management/?utm_source=blankslate&utm_medium=product&utm_content=couponsdoc&utm_campaign=woocommerceplugin">' . esc_html__( 'Learn more about coupons', 'woocommerce' ) . '</a>';
+		echo '<a class="woocommerce-BlankState-cta button-primary button" href="' . get_admin_url() . 'post-new.php?post_type=shop_coupon">' . esc_html__( 'Create your first coupon', 'woocommerce' ) . '</a>';
```

Comment:
Good idea. Thanks. 

Just one small change then I can merge. Can you please make the`admin_url` area on this line [match the same format/style as the ones from other list tables](https://github.com/woocommerce/woocommerce/blob/master/includes/admin/list-tables/class-wc-admin-list-table-products.php#L53). That way everything is consistent and escaped properly across the codebase.

Reply:
Done